examples/research_projects/codeparrot/scripts/preprocessing.py

```diff
@@ -49,18 +49,77 @@ def is_autogenerated(example, scan_width=5):
         return {"autogenerated": False}
 
 
+def is_config_test(example, scan_width=5, max_config=4, max_test=4, max_len=8000):
+    """Check if file is a configuration file or a unit test by :
+         1- looking for keywords in the first few lines of the file
+         2- counting number of occurence of the words 'config' and 'test'.
+    Remark: for the second test we only check files with less than 8000 characters"""
```

Comment:
Do you think we could maybe generalize this to something like number of occurrences per line? That way we would not be so sensitive to the document length and I could imagine that there are very long test files.